tests/decorators/decorators.go

```diff
@@ -60,6 +60,8 @@ var (
 	RequiresNoSnapshotStorageClass = Label("RequiresNoSnapshotStorageClass")
 	// Requires a storage class with ReadWriteMany Block support
 	RequiresRWXBlock = Label("RequiresRWXBlock")
+	// Requires a storage class with ReadWriteOnce Filesystem support
+	RequiresRWOFs = Label("RequiresRWOFs")
```

Comment:
RWO Filesystem is consensus;
If it's block storage, it'll let you put a filesystem on top,
and if it's filesystem storage, it's pointless without single node writing abilities.

Are you sure about introducing such label?

Reply:
Since we are going to be running the kubevirt tests against different storage provisioners, I don't think we need the extra clone set. We should just play well and adjust to what's supplied in the test config

You can safely assume that if we care about a certain storage permutation, it will get tested
So these tests will be exercised for example against
```
{
    "storageClassRhel":        "lvms-vg1",
    "storageClassWindows":     "lvms-vg1",
    "storageRWOFileSystem":    "lvms-vg1",
    "storageRWOBlock":         "lvms-vg1",
    "storageSnapshot":         "lvms-vg1"
}
```

Reply:
I am not sure I am following. The problem is that the `VMStateStorageClass` is either auto-picked or configured by the cluster before test run. So you always want to test only one variant of the `VMStateStorageClass`.

Reply:
@akalenyu PTAL

Reply:
> I can make the label more concrete if it helps
Yeah, that would help, once that is in I can lgtm. I don't think the current label reflect what you're after, and it may or may not confuse folks later. I don't want to block so just let me know.

> I don't think we need the extra clone set
I am saying there's room to generalize those tests to work with what is supplied (and not have to clone entries)
but that's for a different day

Reply:
>Yeah, that would help, once that is in I can lgtm. I don't think the current label reflect what you're after, and it may or may not confuse folks later. I don't want to block so just let me know.

@akalenyu  Any take? 
>I am saying there's room to generalize those tests to work with what is supplied (and not have to clone entries)
but that's for a different day

You are right, back then I remember we wanted explicitly duplicate so user would see that there are multiple choices

Reply:
> @akalenyu Any take?
> 
> >

RequiresNoRWXFSSupport? Depends on what you're after, I think this is it though.